proposals/lip-0042.md

```diff
@@ -43,16 +43,17 @@ In this section, we specify the protocol logic in the lifecycle of a block injec
 
 ### Constants
 
-| **Name**                        | **Type** | **Value**                                 |
-|---------------------------------|----------|-------------------------------------------|
-| `MODULE_NAME_REWARD `           | string   | "reward"                                  |
-| `EVENT_NAME_REWARD_MINTED`      | string   | "rewardMinted"                            |
-| `REWARD_NO_REDUCTION`           | uint32   | 0                                         |
-| `REWARD_REDUCTION_SEED_REVEAL`  | uint32   | 1                                         |
-| `REWARD_REDUCTION_MAX_PREVOTES` | uint32   | 2                                         |
-| `REWARD_REDUCTION_NO_ACCOUNT`   | uint32   | 3                                         | 
-| `TOKEN_ID_REWARD`               | bytes    | specified as part of module configuration |
-| `REWARD_REDUCTION_FACTOR_BFT`   | uint32   | 4                                         |
+| **Name**                        | **Type** | **Value**                                 | **Description**        |
+|---------------------------------|----------|-------------------------------------------|------------------------|
+| `MODULE_NAME_REWARD `           | string   | "reward"                                  | The module name of the Reward module. |
+| `EVENT_NAME_REWARD_MINTED`      | string   | "rewardMinted"                            | Name of the event during minting of the rewards. |
+| `REWARD_NO_REDUCTION`           | uint32   | 0                                         | Return code for no block reward reduction. |
+| `REWARD_REDUCTION_SEED_REVEAL`  | uint32   | 1                                         | Return code for block reward reduction because of the failed seed reveal. |
+| `REWARD_REDUCTION_MAX_PREVOTES` | uint32   | 2                                         | Return code for block reward reduction because the block header does not imply the maximal number of prevotes. |
+| `REWARD_REDUCTION_NO_ACCOUNT`   | uint32   | 3                                         | Return code for block reward reduction because the `TOKEN_ID_REWARD` account is not initialized. |
+| **Configurable Constants** | | | |
+| `TOKEN_ID_REWARD` | bytes | specified as part of module configuration | The [token ID][lip-0051#token-id-and-native-tokens] of the token used for the reward system of the Blockchain. |
+| `REWARD_REDUCTION_FACTOR_BFT` | uint32 | 4 | The reduction factor for validator block reward in case when the block header does not imply the maximal number of prevotes. |
```

Comment:
```suggestion
| `REWARD_REDUCTION_FACTOR_BFT` | uint32 | 4 | The reduction factor for the block reward in case the block header does not imply the maximal number of prevotes. |
```